components/brave_wallet_ui/components/desktop/popup-modals/add-account-modal/hardware-wallet-connect/index.tsx

```diff
@@ -111,6 +109,9 @@ export default function (props: Props) {
           <DisclaimerText>{locale.connectHardwareInfo3}</DisclaimerText>
         </HardwareInfoColumn>
       </HardwareInfoRow>
+      {connectionError !== '' &&
```

Comment:
Added error message to UI cc @Douglashdaniel

Reply:
Looks good!